lib/screens/explore_screens/regions_screens/edit_region/interactor/usecases/edit_region_description_use_case.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:seeds/datasource/local/settings_storage.dart';
+import 'package:seeds/datasource/remote/api/region_repository.dart';
+import 'package:seeds/datasource/remote/model/transaction_response.dart';
+import 'package:seeds/domain-shared/base_use_case.dart';
+
+class EditRegionDescriptionUseCase extends InputUseCase<TransactionResponse, _Input> {
+  final RegionRepository _regionRepository = RegionRepository();
+
+  static _Input input({
```

Comment:
i hate myself for introducing this pattern. i dont like it...

Reply:
I know I also found it weird